Refrigerator/Refrigerator/Program.cs

```diff
@@ -0,0 +1,128 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Refrigerator
+{
+    internal class Program
+    {
+        static void PrintMenu()
+        {
+            Console.WriteLine("Enter 1 to print all the items in the refrigerator and all its contents.\r\n" +
+                "Enter 2 to print how many space is left in the refrigerator.\r\n" +
+                "Enter 3 to put an item in the refrigerator.\r\n" +
+                "Enter 4 to remove an item from the refrigerator.\r\n" +
+                "Enter 5 to clean the refrigerator and print all the checked items.\r\n" +
+                "Enter 6 if you what to eat somthing.\r\n" +
+                "Enter 7 to print all the items sorted by their expiration date.\r\n" +
+                "Enter 8 to print all the shelves sorted according the free space left on them.\r\n" +
+                "Enter 9 to print all the refrigerators sorted according the free space left on them.\r\n" +
+                "Enter 10: to prepare the refrigerator for shopping.\r\n" +
+                "Enter 100 to exit.");
+        }
+
+        static void Main(string[] args)
+        {
+            List<Item> items = Item.InitializeItems();
+            Refrigerator refrigerator = new Refrigerator(ModelType.RF01, RefrigeratorColor.Gray);
```

Comment:
You only create one refrigerator, and you have a sort function, maybe add another refrigerator for showcase?